saleor/graphql/order/mutations/draft_order_create.py

```diff
@@ -117,6 +118,13 @@ def clean_input(cls, info, instance, data):
             object_id=data.pop("shipping_method", None), raise_error=False
         )
 
+        if email := data.get("user_email", None):
+            try:
+                user = User.objects.get(email=email, is_staff=False)
```

Comment:
Is email matching the same in all places we do email matching? For example, is it case-sensitive everywhere? If not, could looking up a user by email be a helper function?

Reply:
Is `is_staff=False` mandatory here? Staff user also could be a customer, right? I think we should check `is_active=True` also.

Reply:
@patrys It looks like it's always case-sensitive, in all places we put `email` data to `get()` `filter()` objects methods directly from the input without any lowercasing, etc.